src/main/java/org/fedoraproject/fedmsg/FedmsgConnection.java

```diff
@@ -40,6 +40,11 @@ public FedmsgConnection connect() {
         return this;
     }
 
+    public boolean disconnect(ZMQ.Socket sock) {
```

Comment:
Rather than passing in a `ZMQ.Socket` here, we probably shouldn't pass any parameters and then use `this.sock.disconnect()`, because this method isn't static and so we already have access to `this.sock`.